### PR TITLE
[Term Entry] JavaScript Dates .setUTCMinutes()

### PR DESCRIPTION
**Pull request by Gpette01**
### Description

- Added the Date.setUTCMinutes() term for JavaScript
- Closes issue #2551 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:3]:
```suggestion
Description: 'Changes the minutes for a Date instance according to universal time (UTC).'
```
The description is of the method, not the article.

File: content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:15]:
```suggestion
The **`.setUTCMinutes()`** method of `Date` instances changes the minutes for this instance according to universal time (UTC).
```
We generally don't link out to parent entries because the Docs navigation provides them automatically.

File: content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:17]:
```suggestion
## Syntax

```

File: content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:23]:
```suggestion

```
Have the parameters be part of the syntax section

File: content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:39]:
```suggestion

- `minutes`: An integer value between 0 and 59 representing the minutes.
- `seconds`: An integer value between 0 and 59 representing the seconds. If `seconds` is specified, you must also specify `minutes`.
- `ms`: An integer value between 0 and 999 representing the milliseconds. If `ms` is specified, you must also specify `minutes` and `seconds`.

```

Comment (SSwiniarski) [reply content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:39]: We do not use HTML in the Markdown. Also this is presented better as a bulleted list.

File: content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:42]:
```suggestion
```
Return value is part of the Syntax section.

File: content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:43]:
```suggestion
The method changes the `Date` instance and returns its new timestamp. If a parameter is `NaN`, the date is set to Invalid Date and `NaN` is returned.
```

File: content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:45]:
```suggestion
## Example

```
No bolding headers, and a blank line comes before and after.

File: content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:46]: Use static dates in the example so the output is consistent.

File: content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:55]:
```suggestion

This example results in the following output.

```

File: content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:56]: Use a static dete in the example so this output will be consistent whenever the example is run.

File: content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:54]:
```suggestion
In the following example, the `.toUTCString()` method converts a `Date` object to a string representation using the UTC(Coordinated Universal Time) timezone:
```
After the edits to this line, move it before the example code block.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Gpette01 I did a first review and left a few comments. Remember to leave a single blank line between headers, paragraphs and code blocks, and do not use HTML in the Markdown.

File: content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md
Comment (Gpette01) [reply content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:15]: I missed this detail of the Docs capability. Thank you.

**Comment by Gpette01** [comment]
@SSwiniarski I will review and fix all the problems with my file and will update the .md file

**Comment by Gpette01** [comment]
@SSwiniarski I have changes the issues with the .md file, I am not sure about the static dates, I have set a static value for the Date instance to be the same with the example i provided. Please let me know if there are more issues with my PR. Thank you.

File: content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:26]:
```suggestion
- `seconds`: An integer value between 0 and 59 representing the seconds. If `seconds` is specified, `minutes` must also be specified.
```
Missed in my first pass. We avoid first- and second-person pronouns in Docs.

File: content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:27]:
```suggestion
- `ms`: An integer value between 0 and 999 representing the milliseconds. If `ms` is specified, `minutes` and `seconds` must also be specified.
```

File: content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:34]:
```suggestion

In the following example, the `.toUTCString()` method converts a `Date` object to a string representation using the UTC(Coordinated Universal Time) timezone.

```
Blank lines before and after.

File: content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:44]:
```suggestion
```
Remove extra blank line.

**Review by SSwiniarski — COMMENTED** [review]
@Gpette01 A few more comments.

**Comment by Gpette01** [comment]
@SSwiniarski Thank you for the feedback, the pronoun, blank lines and needed spacing issues have been fixed. Just to make sure for other possible issues i might try to resolve, linking to parent entries is not needed, linking to other methods inside the docs is okay to add. Is this correct? Thank you.

**Comment by SSwiniarski** [comment]
@Gpette01 That's correct.

**Review by SSwiniarski — APPROVED** [review]
@Gpette01 I did one minor edit and it looks good for a second review.

**Comment by Gpette01** [comment]
@SSwiniarski Perfect thank you very much, I will be waiting for the second review!

File: content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:3]:
```suggestion
Description: 'Changes the minutes for a given date according to universal time (UTC).'
```

File: content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:15]:
```suggestion
The **`.setUTCMinutes()`** method changes the minutes of a `Date` instance according to universal time (UTC).
```

File: content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:19]:
```suggestion
```pseudo
```

File: content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:20]: these lines shouldn't be indented

File: content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:31]:
```suggestion
The method changes the `Date` instance and returns the modified timestamp. If a parameter is `NaN`, the date is invalid and `NaN` is returned.
```

File: content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:48]:
```suggestion
```shell
```

File: content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/setUTCMinutes/setUTCMinutes.md:35]:
```suggestion
In the following example, the `.toUTCString()` method converts a `Date` object to a string representation using the UTC timezone.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Gpette01 I've added my comments. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by Gpette01** [comment]
@caupolicandiaz All recommendations have been issued and I fixed the indented lines in the Syntax section. Thank you.

**Review by caupolicandiaz — APPROVED** [review]
@Gpette01 @SSwiniarski Looks good to merge

**Comment by SSwiniarski** [comment]
@Gpette01 Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/javascript/dates/setUTCMinutes

image

You're currently anonymous. If you don't wish to be anonymous, check out this forum thread on how to link your GitHub and Codecademy profiles: https://discuss.codecademy.com/t/my-contribution-is-live-but-im-not-being-shown-as-a-contributor-why-is-this/758036  **Note:** it may take a few minutes for the change to register on the entry page.

**Pull request merged**
